samples/openthread/ncp/README.rst

```diff
@@ -0,0 +1,88 @@
+.. _ot_ncp_sample:
+
+Thread: NCP sample
+##################
+
+The Thread NCP sample demonstrates the usage of OpenThread Network Co-Processor architecture inside the Zephyr shell.
+
+Overview
+********
+
+The sample demonstrates usage of NCP target on the MCU to communicate it with user WPAN Network Daemon, allowing to create for example Thread Border Router.
+It is important to mention that, according to NCP architecture, MCU part needs to cooperate with user higher layer process to altogether establish complete Thread Border Router.
+Recommended solution is to use docker version of OTBR (OpenThread Border Router) and WPAN Network Daemon tool (wpantund), working under Linux operating system, for such purpose (for more information go there: `OpenThread Border Router`_).
+
+There is possibility to configure number of OpenThread features, but by default application has COMMISSIONER, JOINER, BORDER_ROUTER, BORDER_AGENT and DHCP6_SERVER functionalities enabled.
+
+Requirements
+************
+
+The sample supports the following development kits for testing the network status:
+
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set8_start
+   :end-before: set8_end
+
+Border Router higher layer process is needed to communicate with MCU border router part.
+Optionally, one or more compatible development kits programmed with OpenThread samples to test communication in bigger network.
+
+User interface
+**************
+
+Main recommended interaction with the application should be done via wpantund Linux process using its comannds (for more information check: `wpantund usage`_).
+It is also possible to communicate with board using pyspinel tool via serial interface (for more information check: `pyspinel commands`_).
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/openthread/ncp`
+
+|enable_thread_before_testing|
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+To perform tests it is necessary to have at least one development kit board and second Thread node for checking connection.
+Depending on needs it might be another development kit board running OpenThread stack or the simulated node on PC.
+
+Below steps were based on the given connectivity test: `OpenThread Border Router connectivity test`_, where detailed description may be found.
+
+#. Connect board to the PC and get its serial port name.
+#. Run OTBR using docker and pass board serial port name as ncp interface.
+#. Now bring up second Thread node, for example by flashing development kit board with OpenThread CLI FTD_JOINER sample (see here: `Codelabs - Build Thread network`_)
```

Comment:
Can't we recommend to use NCS CLI sample instead? Should be way simpler in this case than building a sample from OT repo.

Reply:
Done